src/main/webapp/populateData.js

```diff
@@ -89,3 +89,6 @@ function populateTasks() {
       });
 }
 
+function populateGo() {
+  fetch('/directions');
```

Comment:
This does not appear to be populating any information on the panels. Is there any particular reason for calling this?

Reply:
I agree, this was taken from our prototype demo and it was used for demo purposes but it doesn't make sense to retain this here considering the goal is just to have a pipeline for the Directions API. I have removed it, thanks for pointing it out!

Reply:
`RESOLVED`